src/main/java/seedu/address/logic/commands/student/StudentEditCommand.java

```diff
@@ -88,11 +88,6 @@ public CommandResult execute(Model model) throws CommandException {
             throw new CommandException(MESSAGE_DUPLICATE_PERSON);
         }
 
-        if (studentToEdit.belongsTo(editedStudent.getTutorialGroup())
-                && !model.hasTutorialGroup(editedStudent.getTutorialGroup())) {
-            throw new CommandException(MESSAGE_TUTORIAL_GROUP_NOT_FOUND);
-        }
-
```

Comment:
We should probably change the help blurb for edit command also if we're removing this